openslides/core/static/templates/projector.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
-<html lang="{{ selectedLanguage }}" ng-controller="LanguageCtrl" class="no-js">
+<html lang="{{ selectedLanguage }}" ng-controller="LanguageAndFontCtrl" class="no-js">
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <base href="/">
 <title>OpenSlides &ndash; Projector</title>
 <link rel="stylesheet" href="static/css/openslides-libs.css">
 <link rel="stylesheet" href="static/css/openslides-projector.css">
 <link rel="icon" href="/static/img/favicon.png">
+
+<style type="text/css">
```

Comment:
Is it possible to move this style defs to an extra file?

Reply:
I'm not a fan of this, because angular has to replace the `src` there.

Reply:
Ok, this is hacky. Please open an issue to clean up this problem later.

Reply:
Ok.